modules/tensor_mechanics/src/materials/DamageBase.C

```diff
@@ -0,0 +1,54 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "DamageBase.h"
+
+#include "MooseMesh.h"
```

Comment:
I'm not certain why this header is necessary?

Reply:
It's not. Good catch!